types/eslint/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/eslint",
-    "version": "8.44.9999",
+    "version": "8.56.9999",
```

Comment:
Why this field is changed?

Reply:
Hm, it's because the definition has changed in ESLint 8.56.0. There used to be a [header comment](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7e4d18c9f493ea2d9745d355dd4fa3d1f83cc5f4/types/eslint/index.d.ts#L1) with the version information, but I don't see that any more.

Reply:
Thanks for letting me know that!